apps/sms/style/report_view.css

```diff
@@ -113,16 +110,16 @@
   padding-top: 0.9rem;
 }
 
-.report-view > .contact-list div {
+.report-view > .contact-list .network-status {
```

Comment:
Here I wasn't really sure; I thought that we could have a "network-status" class elsewhere. But I think you're right and I can remove ".contact-list".